jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/SpringBootPackagedProcessor.java

```diff
@@ -25,9 +25,17 @@
 public class SpringBootPackagedProcessor implements JarProcessor {
```

Comment:
Looks like the access can be limited to the package.

```suggestion
class SpringBootPackagedProcessor implements JarProcessor {
```